features/proposed/check-domain-mx/README.md

```diff
@@ -0,0 +1,71 @@
+
+Check Domain MX
+===============
+
+As a user signing up for Firefox Accounts,
+I want to receive a clear error message
+when I mistype my email address,
+and the typo'd domain is
+obviously not capable of receiving mail.
+
+## UX Mocks
+
+<img src="no-mx-error.png">
+
+## Success Criteria
+
+We will measure success
+by measuring the rate of email bounces
+(`fxa.auth.account.email_bounced`)
+as a percentage of email verifications
+(`fxa.auth.account.verified`).
+If the feature is providing value
+then we would expect to see
+a decrease in the proportion of emails that bounce,
+since emails that would previously have bounced
+will now be rejected up-front.
+
+The bounce-to-verification rate as of March 2016
+is around 12%:
+
+<img src="initial-bounce-rate.png">
+
+A successull outcome for this feature
+will be to reduce the bounce rate
+by 2% or more.
+
+For an additional measure of usefulness
+we will measure the number of times
```

Comment:
I think more interesting is the % of users that see the message. It's a proxy measure for users that either mistype an email, are being lazy and not adding the TLD, or are just trying to pull one over on us.